.github/workflows/hypersdk-unit-tests.yml

```diff
@@ -8,11 +8,9 @@ on:
     branches:
       - main
   pull_request:
-    types: [labeled,synchronize,reopened]
```

Comment:
why is this line removed?

Reply:
this trigger is only needed when we are listening for labels